src/ispc_version.h

```diff
@@ -20,9 +20,10 @@
 #define ISPC_LLVM_17_0 170000
 #define ISPC_LLVM_18_1 180100
 #define ISPC_LLVM_19_0 190000
```

Comment:
Should the version be `ISPC_LLVM_19_1 190100`?

Reply:
I was going with it forth and back. And I think no.

For `18_1` we have the only usage case `>= 18_1`, which is ok, since `18_0` doesn't exist anymore. But it's really `>= 18_0`.

For `19_0` we have the same usages: `>= 19_0`, which looks right. So I decided not to change it and not to introduce `19_1`.